src/synopsys-action/factory/github-client-service-factory.ts

```diff
@@ -0,0 +1,60 @@
+import {GithubClientServiceInterface} from '../service/github-client-service-interface'
+import {GithubClientServiceCloud} from '../service/impl/cloud/github-client-service-cloud'
+import {info} from '@actions/core'
+import * as constants from '../../application-constants'
+import {GithubClientServiceV1} from '../service/impl/enterprise/v1/github-client-service-v1'
+import {GithubClientServiceV2} from '../service/impl/enterprise/v2/github-client-service-v2'
+import {HttpClient} from 'typed-rest-client/HttpClient'
+import * as inputs from '../inputs'
+
+export const GitHubClientServiceFactory = {
+  SUPPORTED_VERSIONS_V1: ['3.11', '3.12'],
+  SUPPORTED_VERSIONS_V2: ['3.13', '3.14'],
```

Comment:
we just support 3.11 and 3.12, add in v1
we don't need to mention v2, remove everything related to v2

Reply:
removed